ui/src/components/GeneralStatic.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import { FormattedMessage } from "react-intl";
+
+/**
+ * General static component for rendering pagesS
+ *
+ * @class StaticComp
+ * @extends {Component}
+ */
+const StaticComp = ({ subtitle, children }) => (
+  <div>
```

Comment:
We'll probably need a class name to style this div. Could you call it `staticpage-wrapper` or something like that?